webui/react/src/components/ResponsiveTable.tsx

```diff
@@ -86,14 +86,17 @@ const ResponsiveTable: ResponsiveTable = ({
 
   return (
     <div ref={tableRef}>
-      <Spinner spinning={spinning}>
-        <Table
-          bordered
-          scroll={tableScroll}
-          tableLayout="auto"
-          {...props}
-        />
-      </Spinner>
+      {
+        spinning ? <Spinner spinning={true} />
+          : (
+            <Table
+              bordered
+              scroll={tableScroll}
+              tableLayout="auto"
+              {...props}
+            />
+          )
+      }
```

Comment:
question: why is this change needed? 

Overall the preferred pattern in our codebase is to wrap things in `<Spinner></Spinner>` the way it is done before the changes.